pkg/pillar/go.sum

```diff
@@ -1457,6 +1457,8 @@ github.com/lf-edge/edge-containers v0.0.0-20240207093504-5dfda0619b80 h1:kiqB1Rk
 github.com/lf-edge/edge-containers v0.0.0-20240207093504-5dfda0619b80/go.mod h1:4yXdumKdTzF0URMtxOl8Xnzdxnoy1QR+2dzfOr4CIZY=
 github.com/lf-edge/eve-api/go v0.0.0-20250612135640-4c5e51b820c8 h1:muFLH/MPxNylzDK/BcZ5Z/QtnWhd09fJzm5dRi6fLkw=
```

Comment:
Run `go mod tidy` to remove the previous eve-api version

Reply:
Ok, let's rerun the tests. I hope no problems are coming.